### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oliver-walker-hms**
specified which certificate file to put on flexy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# flexy-osisoft
```
Comment (TomKimsey) [inline README.md]: Whatever editor you used added the BOM.  This should be removed.

Comment (oliver-walker-hms) [reply README.md]: It should be removed now

Comment (TomKimsey) [reply README.md]: You will want to remove the change from the first commit.  There are a number of ways to do this in git, the easiest in this case is to probable rebase and apply your 2nd commit as a "fixup" to the first commit.

As a general rule anything introduced in a pull request should not be removed in the same pull request.  The idea being that everything in the pull request is 100% intentional.

Comment (oliver-walker-hms) [reply README.md]: I got it, thanks.
It's definitely good to pick that stuff up now

**Comment by TomKimsey** [comment]
So the next step for you would be to click the "Rebase and merge" button down below, then delete the branch.  This will add these changes to the master branch.  For this repo I have disabled the "Create a merge commit" and "Squash and merge" options, not all repos will have those options disabled.  You want to make sure you always use the "rebase and merge" option.

**Pull request merged**